spock-specs/src/test/groovy/org/spockframework/smoke/parameterization/InvalidWhereBlocks.groovy

```diff
@@ -182,6 +182,28 @@ def foo() {
     "{ -> { -> instanceField }() }()" |_
   }
 
+  def "filter block may not reference instance fields"() {
```

Comment:
I think we need another to test for using local variables.

Reply:
Are you sure?
Actually even this test is questionable.
We already have a test that the `filter` content lands in a static method.
So accessing instance fields as well as local variables is prevented by Groovy already.

Reply:
Nevermind, added.